### DISCUSSION TITLE
Document ID started from huge number

### DISCUSSION THREAD

**Discussion by d47081** [manticoresoftware/manticoresearch-php#163] [Q&A] [answered]
Why document IDs started with the huge integer like `3245002253345489025`?

Is there any solution to make auto-increment index begin from `1` without providing second attribute in `addDocument` method?

Thanks

**Comment by sanikolaev** [comment, marked as answer]
Because it has to include the server ID and timestamp. The detailed explanation can be found in the docs https://manual.manticoresearch.com/Data_creation_and_modification/Adding_documents_to_a_table/Adding_documents_to_a_real-time_table#Auto-ID

**Reply by d47081** [reply]
Thank you!
